src/main/java/com/exactpro/th2/act/ActHandler.java

```diff
@@ -90,6 +97,7 @@
 
 public class ActHandler extends ActImplBase {
     private static final int DEFAULT_RESPONSE_TIMEOUT = 10_000;
+    private static final String SEND_RAW_QUEUE_ATTRIBUTE = "send_raw";
```

Comment:
I think we should add information about this to README file (maybe with an example)